tests/auth.test.js

```diff
@@ -0,0 +1,259 @@
+          email: 'user.unexisted@gmail.com',
+          password: 'Ab1234567&',
+        })
+      )
+      .expect(403);
+  });
+
+  it("Users with wrong password can't login", async () => {
+    await agent
+      .post('/api/auth/login')
+      .set('Content-type', 'application/json')
+      .send(
+        JSON.stringify({
+          email: 'user.one@gmail.com',
+          password: 'Cd8910111&',
+        })
+      )
+      .expect(403);
+  });
+
+  it("Login response don't contain password", async () => {
+    const response = await agent
+      .post('/api/auth/login')
+      .set('Content-type', 'application/json')
+      .send(
+        JSON.stringify({
+          email: 'user.one@gmail.com',
+          password: 'Ab1234567&',
+        })
+      )
+      .expect(200);
+    expect(response.body.password).toBeUndefined();
+  });
+
+  it("Users which isn't active can't login", async () => {
+    // user inactivation by admin
+    const token = signToken({ isActive: true, isAdmin: true });
+    const usersResponse = await agent
+      .get('/api/admin/users')
+      .set('Cookie', [`token=${token}`])
+      .expect(200);
+    const user = usersResponse.body.find((elem) => elem.email === 'user.two@gmail.com');
+    await agent
+      .put('/api/admin/users')
+      .set('Cookie', [`token=${token}`])
+      .set('Content-type', 'application/json')
+      .send(
+        JSON.stringify({ changes: [{ _id: user._id, isAdmin: user.isAdmin, isActive: false }] })
+      )
+      .expect(200);
+
+    // attempt of inactive user to login
+    await agent
+      .post('/api/auth/login')
+      .set('Content-type', 'application/json')
+      .send(
+        JSON.stringify({
+          email: 'user.two@gmail.com',
+          password: 'Ab1234567&',
+        })
+      )
+      .expect(403);
+  });
+});
+
+describe('Test /api/auth/logout path', () => {
+  it('Logout is forbidden without cookies', async () => {
+    await agent
+      .post('/api/auth/logout')
+      .set('Content-type', 'application/json')
+      .send(
+        JSON.stringify({
+          email: 'user.two@gmail.com',
+        })
+      )
+      .expect(403);
+  });
+
+  it('Logout is possible with cookies', async () => {
+    // login to get token for cookies
+    const response = await agent
+      .post('/api/auth/login')
+      .set('Content-type', 'application/json')
+      .send(
+        JSON.stringify({
+          email: 'user.one@gmail.com',
+          password: 'Ab1234567&',
+        })
+      )
+      .expect(200);
+    // logout
+    await agent
+      .post('/api/auth/logout')
+      .set('Content-type', 'application/json')
+      .set('Cookie', [`token=${getCookiesObj(response.headers['set-cookie']).token}`])
+      .send(
+        JSON.stringify({
+          email: 'user.one@gmail.com',
+        })
+      )
+      .expect(201);
+  });
+});
+
+describe('Test /api/auth/username path', () => {
+  it('Getting username withiut cookies is forbidden', async () => {
```

Comment:
@IgorChaikin I suppose it should be `without` instead of `withiut`, also try to use descriptive names for test cases, `Getting username without cookies is forbidden` already exists in same test file